iphone/Classes/TiUIiOSStepper.m

```diff
@@ -0,0 +1,204 @@
+}
+
+#pragma mark backgroundImage
+-(void)setBackgroundImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:YES];
+    [[self stepper] setBackgroundImage:backgroundImageCache forState:UIControlStateNormal];
+}
+
+-(void)setBackgroundSelectedImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:YES];
+    [[self stepper] setBackgroundImage:backgroundImageCache forState:UIControlStateHighlighted];
+}
+
+-(void)setBackgroundDisabledImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:YES];
+    [[self stepper] setBackgroundImage:backgroundImageCache forState:UIControlStateDisabled];
+}
+
+#pragma mark Decrement backgroundImage
+-(void)setDecrementImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setDecrementImage:backgroundImageCache forState:UIControlStateNormal];
+}
+
+-(void)setDecrementSelectedImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setDecrementImage:[self loadImage:value] forState:UIControlStateSelected];
+}
+
+-(void)setDecrementDisabledImage_:(id)value
+{
+    [[self stepper] setDecrementImage:[self loadImage:value] forState:UIControlStateDisabled];
+}
+
+#pragma mark Increment backgroundImage
+-(void)setIncrementImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateNormal];
+}
+
+-(void)setIncrementSelectedImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateSelected];
+}
+
+-(void)setIncrementDisabledImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateDisabled];
+}
+
+-(void)frameSizeChanged:(CGRect)frame bounds:(CGRect)bounds
+{
+    [TiUtils setView:[self stepper] positionRect:bounds];
+    [super frameSizeChanged:frame bounds:bounds];
+}
+
+-(CGFloat)contentWidthForWidth:(CGFloat)value
+{
+    return [[self stepper] sizeThatFits:CGSizeMake(value, 0)].width;
+}
+
+-(CGFloat)contentHeightForWidth:(CGFloat)value
+{
+    return [[self stepper] sizeThatFits:CGSizeMake(value, 0)].height;
+}
+
+-(BOOL)hasTouchableListener
+{
+    // since this guy only works with touch events, we always want them
+    // just always return YES no matter what listeners we have registered
+    return YES;
+}
+
+- (IBAction)stepperChanged:(id)sender
+{
+    NSNumber *newValue = [NSNumber numberWithDouble:[[self stepper] value]];
+    [self.proxy replaceValue:newValue forKey:@"value" notification:YES];
```

Comment:
This is called every time you decrement or increment , I'm not sure what you mean ? Changing the maximum value or minimum shouldn't effect the current value unless the maximum value is smaller than the current value or the minimum is larger. In which case it will change internally and then this will be called. Or if not soon as you increment/decrement it will be updated.